css-fonts-5/Overview.bs

```diff
@@ -93,13 +93,228 @@ Issue(4044):
 <h3 id="font-size-adjust-prop">
 Relative sizing: the 'font-size-adjust' property</h3>
 
-The 'font-size-adjust' property is applied after the 'size-adjust' descriptor.
+	<pre class="propdef">
+	Name: font-size-adjust
+	Value: none | [ ex | cap | ch | ic ]? [ from-font | <<number>> ]
+	Initial: none
+	Applies to: all elements and text
+	Inherited: yes
+	Percentages: N/A
+	Computed value: the keyword ''font-size-adjust/none'', or a map of the specified keywords and their associated <<number>>
+	Animation type: by computed value type
+	</pre>
+
+	For any given font size, the apparent size and effective legibility of text
+	varies across fonts as a function of their design.
+	For example,
+	for bicameral scripts such as Latin or Cyrillic that
+	distinguish between upper and lowercase letters,
+	the relative height of lowercase letters
+	compared to their uppercase counterparts
+	is a determining factor of legibility.
+	In situations where font fallback occurs,
+	fallback fonts might not share
+	the same ratios as the desired font family
+	for key typographic metrics,
+	and will thus appear to be a different size
+	and possibly be less readable.
+
+	The 'font-size-adjust' property provides a way
+	to preserve the readability and apparent size of text
+	when font fallback occurs.
+	It does this by adjusting the used font size
+	so that the specified metric is the same
+	regardless of the font used.
+
+	Values have the following meanings:
+
+	<dl dfn-type=value dfn-for="font-size-adjust">
+		<dt><dfn>none</dfn>
+		<dd>
+			No special 'font-size' adjustment is applied.
+
+		<dt><dfn>ex | cap | ch | ic</dfn>
+		<dd>
+			Specifies the font metric to normalize,
+			defaulting to ''font-size-adjust/ex'':
+
+			<dl>
+				<dt><dfn>ex</dfn>
+				<dd>
+					Normalize the <dfn dfn>aspect value</dfn> of the fonts,
+					using the x-height divided by the font size.
+
+				<dt><dfn>cap</dfn>
+				<dd>
+					Normalize the cap-height of the fonts,
+					using the cap-height by the font size.
+
+				<dt><dfn>ch</dfn>
+				<dd>
+					Normalize the narrow pitch of the fonts,
+					using the advance measure of “0” (ZERO, U+0030)
+					divided by the font size.
+
+				<dt><dfn>ic</dfn>
+				<dd>
+					Normalize the wide pitch of the font,
+					using the advance measure of “水” (CJK water ideograph, U+6C34)
```

Comment:
What if the 水 glyph isn't in the font? Presumably you fallback to `1em` as is the case with the `ic` unit, but since this is defined without reference to that, it's probably worth spelling out.

Reply:
Good point. I should probably just redefine in terms of the units.

Reply:
Actually, that would be bad for ch. Filed issue https://github.com/w3c/csswg-drafts/issues/6384